javascript/javascript1/week1/week-1.js

```diff
@@ -0,0 +1,73 @@
+//* Age-ify (A future age calculator) *//
+
+const yearOfBirth = 2005;
+const yearFuture = 2076;
+
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//* Goodboy-Oldboy (A dog age calculator) *//
+
+const dogYearOfBirth = 2022;
+const dogYearFuture = 2034;
+
+const dogAge = dogYearFuture - dogYearOfBirth;
+shouldShowResultInDogYears = false;
```

Comment:
You forgot to add a _keyword_  at the variable declaration. Specifically, you forgot the `const` or `let` keyword. JavaScript does not give you an error in this case, but without a keyword the variable becomes (global)[https://www.tutorialsteacher.com/javascript/javascript-variable#:~:text=Declare%20Variables%20without%20var%20and%20let%20Keywords&text=The%20variables%20declared%20without%20the,variable%20using%20the%20let%20keyword.], which is soemthing you do not want. Do not forget the `let` or `const` keywords! ☝️